bin/internal/shared.bat

```diff
@@ -88,27 +88,28 @@ GOTO :after_subroutine
   EXIT /B
 
   :do_sdk_update_and_snapshot
-    ECHO Checking Dart SDK version...
-    SET update_dart_bin=%FLUTTER_ROOT%/bin/internal/update_dart_sdk.ps1
+    ECHO Checking Dart SDK version... 1>&2
```

Comment:
Did you double check that this redirects the right thing to the right thing in real life? Line 57/59 does some funny business with the descriptors (see comment in line 58). I almost would have expected a redirect to "3" since that supposedly should be stderr now. Unless of course the file descriptor magic doesn't apply inside the subroutine...

Reply:
Now that I think about this again, the way you have it is correct, I believe. Line 59 redirects stderr "2" of this subroutine to "3", which is stderr in the call context of the subroutine, but not within the subroutine.